test/typescript/testing-matchers/test.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) HashiCorp, Inc
+// SPDX-License-Identifier: MPL-2.0
+import { TestDriver } from "../../test-helper";
+
+describe("typescript testing assertions", () => {
+  let driver: TestDriver;
+
+  beforeAll(async () => {
+    driver = new TestDriver(__dirname);
+    await driver.setupTypescriptProject();
+    await driver.copyFiles("__tests__");
+  }, 6000000);
```

Comment:
1.6 hrs seems a bit excessive 😄 But I can see where this can come from 😁 I think 10 minutes should be enough? 🤔
```suggestion
  }, 600_000);
```